packages/config/src/handlers/locations/children.ts

```diff
@@ -24,5 +24,5 @@ export const resolveChildren: ServerRoute['handler'] = async (req) => {
     return notFound()
   }
 
-  return resolveLocationChildren(location, locations)
+  return [location, ...resolveLocationChildren(location, locations)]
```

Comment:
Metrics was failing due to it sometimes already searching with leaf level id, and then it returned nothing. Now the children search is consistent with the hierarchy search as well, as both also return the current location.